aspnet/security/accconfirm.rst

```diff
@@ -1,236 +1,209 @@
 Account Confirmation and Password Recovery with ASP.NET Identity
 ================================================================
 
-By `Pranav Rastogi`_
+By `Rick Anderson`_
 
 This tutorial shows you how to build an ASP.NET 5 Web site with email confirmation and password reset using ASP.NET Identity.
 
 In this article:
 	- `Create a New ASP.NET 5 Project`_
-	- `Running the Application`_
+	- `Require SSL`_
+	- `Require email confirmation`_
 	- `Setup up Email provider`_
 	- `Enable Account confirmation and Password recovery`_
 	- `Register, confirm email, and reset password`_
 	- `Require email confirmation before login`_
-	- `Next steps`_
-	- `Summary`_
+	- `Combine social and local login accounts`_ 
 
 Create a New ASP.NET 5 Project
 ------------------------------
 
-To get started, open Visual Studio 2015. Next, create a New Project (from the Start Page, or via File - New - Project). On the left part of the New Project window, make sure the Visual C# templates are open and "Web" is selected, as shown:
+Create a new ASP.NET 5 web app with individual user accounts.
 
 .. image:: accconfirm/_static/new-project.png
 
-Next you should see another dialog, the New ASP.NET Project window:
- 
-.. image:: accconfirm/_static/select-project.png
-	
-Select the ASP.NET 5 Web site template from the set of ASP.NET 5 templates. Make sure that authentication is set to **Individual User Accounts**. After selecting, click OK.
+Run the app and then click on the Register link and register a user. At this  point, the only validation on the email is with the `[EmailAddress] <http://msdn.microsoft.com/en-us/library/system.componentmodel.dataannotations.emailaddressattribute(v=vs.110).aspx>`_ attribute. After you submit the registration, you are logged into the app. Later in the tutorial we'll change this so new users cannot log in until their email has been validated.
 
-At this point, the project is created. It may take a few moments to load, and you may notice that Visual Studio's status bar indicates that it is downloading some resources as part of this process. Visual Studio ensures that some required files are pulled into the project when a solution is opened (or a new project is created), and other files may be pulled in at compile time.
+In **SQL Server Object Explorer** (SSOX), navigate to **(localdb)\MSSQLLocalDB(SQL Server 12)**. Right click on **dbo.AspNetUsers** > **View Data**:
 
+.. image:: accconfirm/_static/ssox.png
+.. image:: accconfirm/_static/au.png
 
-Running the Application
------------------------
+Note the ``EmailConfirmed`` field is ``False``.
 
-Run the application and after a quick build step, you should see it open in your web browser.
+Right click on the row and select delete. You'll might want to use this email again in the next step, when the app sends a confirmation email.
```

Comment:
Also, I don't understand the point: "You’ll might want to use this email again in the next step, when the app sends a confirmation email." What do you mean, I "might" want to use this email again?